blog/2024-03-13-magazine-008.md

```diff
@@ -0,0 +1,319 @@
+
+Check out our [getting started tutorial](https://www.winglang.io/docs/) or hit the [Wing Playground](https://www.winglang.io/play/) for an online experience.
+
+### In today's issue
+
+- 🚀 [Deploying AWS resources and APIs inside a private VPCs](#deploying-aws-resources-and-apis-inside-a-private-vpcs)
+- 🎨 [A fresh look for the Wing Console map view](#a-fresh-look-for-the-wing-console-map-view)
+- 🤩 [New syntax for creating sets](#new-syntax-for-creating-sets)
+- 🔥 [Mutually referential type declarations](#mutually-referential-type-declarations)
+- 🛟 [The bang `!` operator](#the-bang--operator)
+- 🏋️ [Ability to lift collection of inflight closures](#ability-to-lift-collection-of-inflight-closures)
+- 🗄️ [`fs.appendFile()`](#fsappendfile)
+- ⚙ [Platform Parameters and AWS VPC configuration](#platform-parameters-and-aws-vpc-configuration)
+- 🛌 [Ergonomic improvements for `cloud.Function` and `cloud.Api`](#ergonomic-improvements-for-cloudfunction-and-cloudapi)
+- 🎥 [Goodies from the Wingly Update](#goodies-from-the-wingly-update)
+- 👯‍♀️ [Community Events](#community-events)
+
+> NOTE: I spent like two hours trying to figure out the right emojis here so please take a minute to appreciate them :-)
+
+### Deploying AWS resources and APIs inside a private VPCs
+
+![](./assets/2024-03-13-magazine-008/private-api.png)
+
+Users tell us they are looking for ways to deploy API-based services inside their VPCs. We've spent some time learning about [the challenges people have](https://www.winglang.io/blog/2024/02/15/private-api-gateways) and figured we can offer a awesome experience in Wing for this use case.
+
+We published a [guide](https://www.winglang.io/blog/2024/02/09/private-api-gateway-aws) which describes how to configure the `tf-aws` platform to deploy your application's AWS resources and APIs inside a VPC, with support for creating a whole new VPC or deploying into a VPC that's already setup in your environment.
+
+Check out a recording of this [online workshop](https://youtu.be/1o5dk4S82_o?si=mvHGaYBJtGTj2Fi8) by [Hasan Abu-Rayyan](https://github.com/hasanaburayyan) from the Wing core team, where he talks about this use case and walks through an example.
+
+[Ruslan Kazakov](https://github.com/rkazakov) from Sydney, Australia wrote about his experience in [a blog post](https://medium.com/bear-plus/secure-your-vpc-endpoint-for-api-gateway-with-winglang-235a5f8af072) :
+
+> ”Wing made all the difference! After struggling for days with the CDK, I turned to Wing and had a fully functional Private API Gateway up and running within hours. Its simplicity and power are unparalleled. Thank you, Wing, for saving me time and headaches!”
+
+### A fresh look for the Wing Console map view
+
+We gave the map view in Wing Console some love, improved colors, cleaned up the UI and polished the interactions:
+
+<ReactPlayer width="50%" height="50%" playing loop muted url={console_new_look}/>
```

Comment:
Nothing is displayed...

Reply:
See [preview](https://wing-docs-git-eladb-inflight-magazine-7-monada.vercel.app/blog/2024/03/13/magazine-008)